api/resolver/mutation/project/project.go

```diff
@@ -0,0 +1,46 @@
+package project
+
+import (
+	"context"
+	"github.com/robojones/graphql/gqlgen"
+	"github.com/robojones/graphql/lib/session_context"
+	"github.com/robojones/graphql/prisma"
+)
+
+type ProjectResolver struct {
```

Comment:
Naming it just `Project` would be fine, wouldn't it?

Reply:
I think I did that because it made sense when I tried another structure for the project. I will change it and remove "-Resolver" from all the struct names.

Reply:
:+1: it will be way more readable that way